addons/tmf_loadouts/loadouts/US_USMC_1985.hpp

```diff
@@ -0,0 +1,493 @@
+// If you want to base a loadout on an existing one, this repository contains them all:
+// https://github.com/ARCOMM/arc_misc/tree/master/addons/tmf_loadouts
+
+//Dependencies:
+//		* CUP Units
+//		* CUP Weapons
+//		* FIR AWS(AirWeaponSystem)
+//		* USM Uniforms
+//		* HLC M21
+
+class baseMan {// Weaponless baseclass
+	displayName = "Unarmed";
+	// All randomized.
+	uniform[] = {
+		LIST_3("CUP_U_B_BDUv2_roll2_dirty_desert"),
+		"CUP_U_B_BDUv2_dirty_desert"
+	};
+	vest[] = {
+		"usm_vest_pasgtdes_lbe_rm_m"

```

Comment:
Mixed CUP and USM uniforms/vests will clip together. Use either of them, but don't mix.

Reply:
Swapped to USM as the camoflage, despite being brighter in the arsenal, appears to match the in-game landscapes and lighting better.